bin/oxfordshire/send-rdi-emails

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use v5.14;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use DateTime;
+use Try::Tiny;
+use FixMyStreet;
+use FixMyStreet::Cobrand;
+use FixMyStreet::Email;
+use FixMyStreet::Integrations::ExorRDI;
+
+my $end_date = DateTime->now( time_zone => FixMyStreet->time_zone || FixMyStreet->local_time_zone )
+    ->truncate(to => 'hour')->set_hour(16);
+my $start_date = $end_date->clone->subtract(days => 1);
+
+my $cobrand = FixMyStreet::Cobrand->get_class_for_moniker('oxfordshire')->new;
+my @inspectors = $cobrand->users->search({
+    'user_body_permissions.permission_type' => 'report_inspect'
+}, {
+    join => 'user_body_permissions',
+    distinct => 1,
+})->all;
+
+foreach my $inspector (@inspectors) {
```

Comment:
That's the spec: https://github.com/mysociety/fixmystreetforcouncils/issues/183#issuecomment-297329233